### PR TITLE
Add cosmetics store, moisturizer

### PR DESCRIPTION
**Pull request by snipercup**
#### Summary
Content "Add stand-alone cosmetics store, moisturizer"

#### Purpose of change

Wanted to add a stand-alone cosmetics store

#### Describe the solution

Add a new terrain, s_cosmetic and a new item, moisturizer
Moisturizer doesn't do anything and cannot be consumed.

#### Describe alternatives you've considered
Give an effect to moisturizer but I don't know what.

#### Testing
Spawned the cosmetics store + roof, everything works as expected

#### Additional context

image
image

image

Ammo type soap:
image

### CODE REVIEW

**Comment by fairyarmadillo** [comment]
The moisturizer could be used as ammunition for shave kits, the same way soap is currently. It makes a pretty decent shaving cream IRL.

For a more far in the future thing, it'd be neat to be able to use makeup to temporarily hide certain mutations and offset their ugliness.

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -756,6 +756,18 @@
...
   },
+  {
+    "id": "moisturizer",
+    "type": "GENERIC",
+    "name": { "str": "moisturizer" },
+    "description": "A cosmetic preparation used for protecting, moisturizing, and lubricating the skin.",
+    "volume": "50ml",
+    "weight": "50 g",
+    "price": 300,
+    "price_postapoc": 300,
```
Comment (cainiaowu) [inline data/json/items/comestibles/med.json:767]: If it is that valuable post apoc, maybe give it some use? Like a small morale boost if "consumed" or something.

Comment (snipercup) [reply data/json/items/comestibles/med.json]: It's now worth as much as soap

**Comment by Gotdamnmiracle** [comment]
Moisturizer could reduce the player's scent for zombies by masking it. They're trying to smell out a living human- not lavender and sandlewood.

**Comment by snipercup** [comment]
> Moisturizer could reduce the player's scent for zombies by masking it. They're trying to smell out a living human- not lavender and sandlewood.
> 
> Assuming it is scented.
> 
> Additionally, you could also make a recipe for it using lanolin (if you add it as part of the sheep shearing process) and flowers (of which I only think lily is an object in the game, the rest are seeds and fruit).

There's a whole scent discussion in #48749 that I'm not getting into right now but I am in favor of masking scent somehow. Adding it to a recipe may be an option for a future PR but I'd consider out of scope for now.

**Pull request merged**